selfdrive/car/hyundai/values.py

```diff
@@ -292,6 +292,19 @@ class Buttons:
       [HYUNDAI_VERSION_REQUEST_MULTI],
       [HYUNDAI_VERSION_RESPONSE],
     ),
+    # CAN-FD queries
+    Request(
+      [HYUNDAI_VERSION_REQUEST_LONG],
+      [HYUNDAI_VERSION_RESPONSE],
+      whitelist_ecus=[Ecu.fwdRadar],
+      bus=4,
```

Comment:
E-CAN is bus 5, and from the schematics the radar is not on E-CAN, so this checks out why it was skipping on bus 5, but not A-CAN (4/6).

It also looks like the ADAS Unit is on E-CAN as well

Reply:
The ADAS ECU (Driving) is not available on bus 4, but it doesn't skip on bus 5 (A-CAN), so using that.